process/block/sovereignChainBlock_test.go

```diff
@@ -61,7 +66,11 @@ func TestSovereignBlockProcessor_NewSovereignChainBlockProcessorShouldWork(t *te
 
 		arguments := CreateMockArguments(createComponentHolderMocks())
 		sp, _ := blproc.NewShardProcessor(arguments)
-		scbp, err := blproc.NewSovereignChainBlockProcessor(sp, nil)
+		scbp, err := blproc.NewSovereignChainBlockProcessor(blproc.ArgsSovereignChainBlockProcessor{
+			ShardProcessor:               sp,
+			ValidatorStatisticsProcessor: nil,
+			OutgoingOperationsFormatter:  &sovereign.OutgoingOperationsFormatterStub{},
```

Comment:
please add a test for OutgoingOperationsFormatter = nil

Reply:
Already added test in next PR here: https://github.com/multiversx/mx-chain-go/pull/5673/files#diff-c6abeee02153644592d72054a78cc21503eb6607028caaf7df1a79c364ba59b8R110